### PR TITLE
AnigliScans connector Added!

### LINKED ISSUE #5327: [Site Request] Animated Glitched Scans
**Name of the website**
Animated Glitched Scans

**Website urls (examples below)**
- Site: https://anigliscans.com/
- Manga List: https://anigliscans.com/series/
- Manga example: https://anigliscans.com/series/monsters/
- Chapter online viewer example: https://anigliscans.com/monsters-chapter-14/

**Languages**
English
List of languages supported by the website

**Additional details**
Similar to Realm Scans. Can use similar connector :)
Add any other context details that you may have found (like template to reuse)

### PR DESCRIPTION
**Pull request by abhishek-exists**
Fixes #5327 
Please verify the code. Tell me if something is wrong. Its my first Open-source contribution 🙂.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "@logtrine/logtrine": "latest",
     "asar": "latest",
     "discord-rpc": "latest",
-    "electron": "8.3.4",
+    "electron": "^8.5.5",
```
Comment (MikeZeDev) [inline package.json]: Delete this change. Its useless.

File: src/web/mjs/connectors/AniGliScans.mjs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class AniGliScans extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'anigliscans';
+        super.label = 'AniGliScans';
+        this.tags = [ 'manga', 'english' ];
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/AniGliScans.mjs]: Its a label, you should display the full website name here.

Comment (MikeZeDev) [reply src/web/mjs/connectors/AniGliScans.mjs]: Also, you may want to add 'scanlation' in tags if its a translator group website.

File: src/web/mjs/connectors/AniGliScans.mjs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        this.url = 'https://anigliscans.com';
+        this.path = '/series/?list';
+    }
+    async _getPages(chapter) {
+        const data = await super._getPages(chapter);
+        return data.map(element => this.createConnectorURI(element));
+    }
+    async _handleConnectorURI(payload) {
+        let request = new Request(payload, this.requestOptions);
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/AniGliScans.mjs]: Its good. You could change let for const as their values are not changed after.

**Review by MikeZeDev — CHANGES_REQUESTED** [review]
* A few improvements possible
* Do not change electron version.
* Icon name must be lowercase and match connector id
* You can link issues in your PR messages with 'Fixes <link>' so the issue will be closed  automatically when its merged 🗡️

File: src/web/mjs/connectors/AniGliScans.mjs (hunk 3 above)
Comment (abhishek-exists) [reply src/web/mjs/connectors/AniGliScans.mjs]: Done, will update the changes

**Comment by abhishek-exists** [comment]
I have done the changes you requested, review it.

**Comment by MikeZeDev** [comment]
Icon name still needs to be lowercase.

**Comment by Sheepux** [comment]
to change the casing, first change completely the name then commit, then rename it to correct lowercase then commit again. Trying to directly change the cast doesn't work.

**Comment by abhishek-exists** [comment]
@Sheepux Thanks. I was busy at that time. I have updated it.

**Review by MikeZeDev — APPROVED** [review]

**Comment by 27136773** [comment]
[Missing in HaruNeko](https://github.com/search?q=repo%3Amanga-download%2Fharuneko%20%27anigliscans%27&type=code)

**Pull request merged**